LICENSE

```diff
@@ -1,13 +1,12 @@
-Fingerprintjs2 – Modern & flexible browser fingerprint library v2
+FingerprintJS – Modern & flexible browser fingerprint library
 https://github.com/fingerprintjs/fingerprintjs
-Copyright (c) 2018– Jonas Haag (jonas@lophus.org)
-Copyright (c) 2015– Valentin Vasilyev (valentin.vasilyev@outlook.com)
```

Comment:
@Valve I don't know whether it's fare to make such changes, so please review

Reply:
looks good, as v3 will be a full rewrite. We can keep Jonas for v2 and remove in v3.

Reply:
Ok, I'll leave him together with FingerprintJS Inc until v3